src/scripts/main.js

```diff
@@ -354,7 +354,26 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+function createTableRow(tableId) {
+  people.age = people.born - people.died;
+  people.century = Math.ceil(people.died / 100);
 
-// write your code here
+  const table = document.querySelector('dashboard');
```

Comment:
The 'querySelector' method is missing a '#' or '.' to select an element by ID or class. It should be 'document.querySelector('#dashboard')' or 'document.querySelector('.dashboard')' depending on whether 'dashboard' is an ID or class.